VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -403,7 +403,7 @@ public void setTitle(CharSequence title) {
                 if(titleTextView!=null){
                     titleTextView.setTextColor(getResources().getColor(R.color.edx_white));
                     titleTextView.setTypeface(type);
-                    bar.setTitle(title);
+                    bar.setTitle("  " + title);
```

Comment:
@aleffert leading space is part of the text, and in RTL, the text will be reverted? so it should be fine.